.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: File",
+            "type": "python",
+            "request": "launch",
+            "program": "${file}",
+            "justMyCode": true
+        },
```

Comment:
what's the purpose of this section (i.e. l:4-10) it cannot directly be used for debugging right?

Reply:
Yes. You can use it for debugging any file by adding the `if __name__ == "__main__": ...` line. But I'll remove it as I infer you don't want to debug any file that way.